packages/config/jest.setup.js

```diff
@@ -0,0 +1,3 @@
+global.afterEach(() => {
+    document.body.innerHTML = '';
```

Comment:
I wonder if there is a method in the jsdom environment that can be called instead of clearing the document body like this?

Reply:
we can regenerate the dom entirely, but in general it's probably overkill and is resource intensive compared to just this method (especially since we have multiple tests)